ModularTegustation/ego_weapons/melee/zayin.dm

```diff
@@ -328,8 +328,9 @@
 	icon_state = "eclipse"
 	force = 14
 	damtype = WHITE_DAMAGE
-	attack_verb_continuous = list("smacks", "strikes", "beats")
-	attack_verb_simple = list("smack", "strike", "beat")
+	attack_verb_continuous = list("slices", "slashes", "stabs")
+	attack_verb_simple = list("slices", "slashes", "stabs")
```

Comment:
i love bad copy-paste, i love bad copy-paste, i love bad copy-paste (merging when this is resolved since its a small change)
```suggestion
	attack_verb_simple = list("slice", "slash", "stab")
```